substrate-node/pallets/pallet-smart-contract/src/tests.rs

```diff
@@ -1639,7 +1646,7 @@ fn test_rent_contract_canceled_due_to_out_of_funds_should_cancel_node_contracts_
         // Event 18: Node contract canceled
         // Event 19: Rent contract Canceled
         // => no Node Contract billed event
-        assert_eq!(our_events.len(), 20);
+        assert_eq!(our_events.len(), 31);
```

Comment:
If I follow the logic, the 11 events difference is because of  

```rust
let twin_balance = Self::get_usable_balance(&twin.account_id);
```
right?

Reply:
I suppose it is. Can we maybe check them to be sure? I would also suggest to remove the comments mentioning the events or update them.